diff --git a/heat/version.py b/heat/version.py
index a717606..8ea213a 100644
--- a/heat/version.py
+++ b/heat/version.py
@@ -14,6 +14,14 @@
 #    under the License.
 
 
-import pbr.version
+class VersionInfo(object):
+    release = 'REDHATHEATRELEASE'
+    version = 'REDHATHEATVERSION'
 
-version_info = pbr.version.VersionInfo('heat')
+    def version_string(self):
+        return self.version
+
+    def release_string(self):
+        return self.release
+
+version_info = VersionInfo()
